sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -302,11 +302,17 @@ impl SfntFont {
             let offset: u64 = entry.offset as u64;
             let size: usize = entry.length as usize;
             // Create a table instance for it.
-            let table: Table = {
+            let table: NamedTable = {
                 match entry.tag {
-                    C2PA_TABLE_TAG => Table::C2PA(TableC2PA::from_reader(reader, offset, size)?),
-                    HEAD_TABLE_TAG => Table::Head(TableHead::from_reader(reader, offset, size)?),
-                    _ => Table::Unspecified(TableUnspecified::from_reader(reader, offset, size)?),
+                    C2PA_TABLE_TAG => {
```

Comment:
I would suggest DRYing this up as well and provide a helper method in a `impl NamedTable`, maybe like so:

```rust
impl NamedTable {
    /// Reads a new instance from the given source.
    pub(crate) fn from_reader<T: Read + Seek + ?Sized>(
        tag: &SfntTag,
        reader: &mut T,
        offset: u64,
        length: usize,
    ) -> Result<Self> {
        match *tag {
            C2PA_TABLE_TAG => Ok(NamedTable::C2PA(TableC2PA::from_reader(
                reader,
                offset,
                length,
            )?)),
            HEAD_TABLE_TAG => Ok(NamedTable::Head(TableHead::from_reader(
                reader,
                offset,
                length,
            )?)),
            _ => Ok(NamedTable::Unspecified(TableUnspecified::from_reader(
                reader,
                offset,
                length,
            )?)),
        }
    }
}
```

And then could clean up here and in woff with:

```rust
let table = NamedTable::from_reader(&entry.tag, reader, offset, size)?;
```

Reply:
Done and done